packages/manager/apps/pci-public-ip/src/components/input/RegionInput.component.tsx

```diff
@@ -36,7 +36,9 @@ export const RegionInputComponent = ({
   onInput,
 }: TProps): JSX.Element => {
   const { me } = useMe();
-  const [selectedMacroName, setSelectedMacroName] = useState<string>(null);
+  const [selectedMacroName, setSelectedMacroName] = useState<string>(
```

Comment:
if `selectedMacroName` could be instantiated with `null`, type should be updated accordingly: 
```suggestion
  const [selectedMacroName, setSelectedMacroName] = useState<string | null>(
```